### PR TITLE
NEOS-116: Transformersv0

### PR DESCRIPTION
**Pull request by evisdrenova**

### CODE REVIEW

File: worker/pkg/workflows/datasync/activities.go [hunk 1]
```diff
@@ -313,6 +314,8 @@ func (a *Activities) Sync(ctx context.Context, req *SyncRequest) (*SyncResponse,
 
 	streambldr := service.NewStreamBuilder()
 
+	fmt.Println("benthos config", req.BenthosConfig)
```
Comment (nickzelei) [inline worker/pkg/workflows/datasync/activities.go:317]: let's get the printlns out of here

File: worker/pkg/workflows/datasync/activities.go [hunk 2]
```diff
@@ -432,95 +436,15 @@ func buildProcessorMutation(cols []*mgmtv1alpha1.JobMapping) (string, error) {
 }
 
 func computeMutationFunction(transformer string) (string, error) {
+
+	fmt.Println("the transfomer in compute", transformer)
```
Comment (nickzelei) [inline worker/pkg/workflows/datasync/activities.go:440]: same here

File: worker/pkg/workflows/datasync/activities.go [hunk 3]
```diff
@@ -432,95 +436,15 @@ func buildProcessorMutation(cols []*mgmtv1alpha1.JobMapping) (string, error) {
...
 	switch transformer {
-	case "uuid_v4":
```
Comment (nickzelei) [inline worker/pkg/workflows/datasync/activities.go:436]: can we bring all of these back?

Comment (evisdrenova) [reply worker/pkg/workflows/datasync/activities.go:436]: yup can add them back

**Review by nickzelei — APPROVED** [review]

**Pull request merged**